tools/make.py

```diff
@@ -697,11 +697,9 @@ def version_stamp_pboprefix(module,commitID):
         f.close()
 
         if configtext:
-            patchestext = re.search(r"version.*?=.*?$", configtext, re.DOTALL)
-            if patchestext:
+            if re.search(r"version=(.*?)$", configtext, re.DOTALL):
```

Comment:
Isn't it opposite? When you use the multiline flag dollar sign is used for end of string, without it end of line. I also verified it worked correctly, in old pboprefix files there were more lines after version and they were never picked up.